src/app/ui/components/dowpdown-menu/dropdown-menu.tsx

```diff
@@ -22,41 +23,43 @@ const dropdownTriggerStyles = css({
   px: 'space.04',
   py: 'space.03',
   textStyle: 'label.02',
+  userSelect: 'none',
+  '[data-state=open] &': { bg: 'ink.component-background-pressed' },
+});
+function Button({ children, ...props }: HTMLStyledProps<'div'>) {
```

Comment:
@kyranjamie rebased on these changes and slightly confused bc this doesn't appear to be used? You coming back to it?

Reply:
Nvm, I see what is happening here, you used the 'unstyled' trigger in code and left this for use as the styled button. I am working on fixing the settings menu trigger button in Pete's work, so in this code a bit.